src/main/resources/org/sonatype/nexus/ci/iq/Messages.properties

```diff
@@ -30,6 +30,7 @@ IqPolicyEvaluation.FailOnNetwork=Fail build when unable to communicate with IQ S
 IqPolicyEvaluation.JobSpecificCredentials=Use job specific credentials
 
 IqPolicyEvaluation.NoIqServersConfigured=No IQ Server configured.
+IqPolicyEvaluation.AutoAppMissing=Missing Auto App Creation.  Possible version mismatch. Please check if the IQ Server is compatible with this plugin version.  Supported IQ server versions are 1.45.0 or newer.
```

Comment:
Minor: Extra space between the first period and `Possible`.

Reply:
>Minor: Extra space between the first period and Possible.
Minor: Extra space between the period after version and Supported.



Changed 2580539